framework/log/LoggerInterface.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
```

Comment:
phpstorm garbage. I disable this thing in settings always.

Reply:
damn it. I should also do it. I've removed that notice and added the yii copyright notice!